content/blog/introducing-os1.md

```diff
@@ -0,0 +1,141 @@
+
+OS 1 is just a client for Urbit OS, of course. Other people could write their own clients that are completely different — like different windowing systems for Linux. Our hope is to build a client that includes interface components and frameworks that make it exceptionally easy to build modules for groups of people to share. Other clients could potentially invent entirely new ways of interacting with the underlying system.
+
+There are two threads by which OS 1 evolves into OS 2 and beyond. First, by becoming a more powerful system and second, by adding modules to expand our supported modes for interacting. 
+
+In its current form, OS 1 by no means exposes all the power of the underlying system. Our approach to building OS 1 was to focus on the quality of the UI, and accept that it will feel like a monolithic app. We’re already working to evolve our interface into something that feels flexible to an ordinary user and developers can easily extend and build on top of. 
+
+Specifically, we want better affordances for putting multiple streams of data on the screen at once, better keyboard shortcuts and navigation affordances, better libraries and frameworks for building modules. We want a standalone client and a single login system. These are some of the things currently on the drawing board.   
+
+As far as modules are concerned, there are too many things we’d like to build to list them all. We’d really like to have threaded conversations; short, Twitter-style messages; biometric and status tracking; an e-book reader and annotations system; and that’s just to name a few we see in the near term. 
+
+We’ve long since felt that the most powerful thing about Urbit is that it’s a general-purpose system. OS 1 puts us on the road to giving people an actual material sense of what using a general-purpose networked computer feels like. 
+
+As we expand the affordances of the system and add modules over the course of OS 1, 2 and beyond we expect to deliver a system that can foster everyday creativity. There’s a long road ahead, in the best possible sense. We’re really excited about what the future holds. 
+
+### Becoming a homesteader
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/intro-os1-landscape.png">
+
+As a distributed team trying to stay connected, we like using OS 1 from day to day. We’d really like to hand onboard a small group of like minded groups of people who want to experiment with living on the fringe of the digital world. Toward this end, we started the community grants program.
+
+Each community that’s selected will get a block of address space to share, direct support and (in the near future) free hosting. 
+
+Please tell us a bit about yourselves [here]().
```

Comment:
link to community grants i believe, lives here: https://urbit.org/community/community-grants/